From ef7945d2295d9b9ed07511b5eb7efa04f5019b3d Mon Sep 17 00:00:00 2001
From: Steev Klimaszewski <steev@gentoo.org>
Date: Sun, 19 Aug 2012 03:11:10 -0500
Subject: [PATCH] Add check for lm since we use maths

Since we use maths, we need to add -lm in order for linking to occur properly with gold.

Signed-off-by: Steev Klimaszewski <steev@gentoo.org>
---
 configure.ac |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/configure.ac b/configure.ac
index 6aae24e..5941acf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -455,6 +455,8 @@ fi
 
 AM_CONDITIONAL([ENABLE_PYTHON],[test "x$enable_python" = "xyes"])
 
+AC_SEARCH_LIBS([floor],[m])
+
 AC_CONFIG_FILES([
 Makefile
 src/Makefile
-- 
1.7.3.4

